example.config.toml

```diff
@@ -1,6 +1,11 @@
+# All fields are required for now
 [general]
+token_size = 123
 # Telegram ID of the master account
 masterid = 1234578
 
 [database]
+host = "0.0.0.0"
 password = "password"
+username = "spam-watch"
+name = "spam-watch-db"
```

Comment:
Should be `SpamWatchAPI`  to be specific and I don't think it's necessary to have the DB suffix as it's implied that it's a Database